tests/lua/test_standalone.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+# Copyright (c) GitHub, Inc.
+# Licensed under the Apache License, Version 2.0 (the "License")
+
+set -xe
+cd "src/lua"
```

Comment:
With the latest addition of packaging, bcc-lua can come either from a package or in-tree. Both should continue to work. Probably what you want is like we do with the wrapper.sh, which is to update the PATH accordingly, and here just check for `which bcc-lua`.